cmd/glasskube/cmd/uninstall.go

```diff
@@ -64,16 +67,39 @@ var uninstallCmd = &cobra.Command{
 					}
 				}
 			}
+		} else if uninstallCmdOptions.DeleteNamespace {
+			// list all packages in the namespace
+			lister := list.NewListerWithRepoCache(ctx)
+			namespace := pkg.GetNamespace()
+			pkgsInNamespace, err := lister.GetPackagesWithStatus(ctx, list.ListOptions{
+				Namespace: namespace,
+			})
+			if err != nil {
+				fmt.Fprintf(os.Stderr, "❌ Could not list packages: %v\n", err)
+				cliutils.ExitWithError()
+			}
+			pkgCount := 0
+			for _, p := range pkgsInNamespace {
+				pkgCount += len(p.Packages)
+			}
+			if pkgCount > 1 {
+				fmt.Fprintf(os.Stderr, "❌ Namespace %v contains more than one package. "+
+					"Current count of packages present in the namespace: %v\n", namespace, pkgCount)
+				cliutils.ExitWithError()
+			}
+			deleteNamespace = true
```

Comment:
This seems like a lot of unnecessary work. It would be better to use a simple packageClient and just fetch the Package resources directly, rather than using the list package, since that one goes through some extra hoops.

Reply:
Makes sense. Fixed it